### PR TITLE
[DNM-1109] added support for php disbursement

### PR DESCRIPTION
**Pull request by hglohika**

### CODE REVIEW

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.Arrays;
+
+import com.xendit.XenditClient;
+import com.xendit.model.DisbursementPHP;
+
+public class ExampleGetPHPDisbursement {
+    public static void main(String[] args) {
+        // create xendit client which holds value of apikey
+        XenditClient xenditClient = new XenditClient.Builder()
+                .setApikey("xnd_development_nsn3PhwpWyMZeCZ9YZNlzh5TVmGTqYxVBoXpdbz2glHi1Nk1dBZOibZdUmqw7")
```
Comment (xenditjoshjimenez) [inline xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java:14]: @hglohika please put a dummy for this one instead.

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ExampleCreatePHPDisbursement {
+    public static void main(String[] args) {
+
+        // create xendit client which holds value of apikey
+        XenditClient xenditClient = new XenditClient.Builder()
+                .setApikey("xnd_development_nsn3PhwpWyMZeCZ9YZNlzh5TVmGTqYxVBoXpdbz2glHi1Nk1dBZOibZdUmqw7")
```
Comment (xenditjoshjimenez) [inline xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java:19]: same here

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+             */
+            Beneficiary beneficiary = Beneficiary.builder()
+                    .type("INDIVIDUAL")
+                    .givenNames("Test Name")
+                    .middleName("Middle Name")
+                    .surname("Sur Name")
+                    .businessName("Test")
+                    .streetLine1("Jl. 123")
+                    .streetLine2("Jl. 456")
+                    .city("Jakarta Selatan")
```
Comment (xenditjoshjimenez) [inline xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java:96]:
```
                   .city("City")
                   .province("Province")
```

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.util.Arrays;
+
+import com.xendit.Xendit;
+import com.xendit.model.DisbursementPHP;
+
+public class ExampleGetPHPDisbursement {
+    public static void main(String[] args) {
+        // access key with Option
+        Xendit.Opt.setApiKey("xnd_development_CnXTAQWxG5qax2csldXWa65PvRAH3XOnqrtTMi5RPyQBUhdqfLcj2vEaFHGUk7Zv");
```
Comment (xenditjoshjimenez) [inline xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java:13]: same here.

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+        try {
+            /**
+             * Get disbursement object by ID.
+             */
+            String disbursementId = "614acbe0c0041e00247ad195";
```
Comment (xenditjoshjimenez) [inline xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java:19]: same here.

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+                                        .businessName("Test")
+                                        .streetLine1("Jl. 123")
+                                        .streetLine2("Jl. 456")
+                                        .city("Jakarta Selatan")
+                                        .province("DKI Jakarta")
+                                        .state("Test")
+                                        .country("Test")
+                                        .zipCode("12345")
+                                        .mobileNumber("9876543210")
+                                        .phoneNumber("987654321")
```
Comment (xenditjoshjimenez) [inline xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java:99]: can add sample country code for this? `+63`

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+                .build();
+
+        try {
+            /**
+             * Get disbursement object by ID.
+             */
+            String disbursementId = "614acbe0c0041e00247ad195";
```
[the remark this answers is not in the document]
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java (hunk 1 above)
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetPHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetDisbursementChannels.java [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+import com.xendit.exception.XenditException;
+import com.xendit.XenditClient;
+import com.xendit.model.DisbursementChannel;
+
+public class ExampleGetDisbursementChannels {
+    public static void main(String[] args) {
+        // create xendit client which holds value of apikey
+        XenditClient xenditClient = new XenditClient.Builder()
+                .setApikey("xnd_development_nsn3PhwpWyMZeCZ9YZNlzh5TVmGTqYxVBoXpdbz2glHi1Nk1dBZOibZdUmqw7")
```
[the remark this answers is not in the document]
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithClient/ExampleGetDisbursementChannels.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java (hunk 2 above)
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java (hunk 3 above)
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithClient/ExampleCreatePHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java (hunk 4 above)
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java (hunk 5 above)
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetPHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetDisbursementChannels.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
+package ExampleWithoutClient;
+
+import com.xendit.exception.XenditException;
+import com.xendit.Xendit;
+import com.xendit.model.DisbursementChannel;
+
+public class ExampleGetDisbursementChannels {
+    public static void main(String[] args) {
+        // access key with Option
+        Xendit.Opt.setApiKey("xnd_development_CnXTAQWxG5qax2csldXWa65PvRAH3XOnqrtTMi5RPyQBUhdqfLcj2vEaFHGUk7Zv");
```
[the remark this answers is not in the document]
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleGetDisbursementChannels.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+                        Beneficiary beneficiary = Beneficiary.builder()
+                                        .type("INDIVIDUAL")
+                                        .givenNames("Test Name")
+                                        .middleName("Middle Name")
+                                        .surname("Sur Name")
+                                        .businessName("Test")
+                                        .streetLine1("Jl. 123")
+                                        .streetLine2("Jl. 456")
+                                        .city("Jakarta Selatan")
+                                        .province("DKI Jakarta")
```
[the remark this answers is not in the document]
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

File: xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java (hunk 6 above)
Comment (hglohika) [reply xendit-java-library-example/src/main/java/ExampleWithoutClient/ExampleCreatePHPDisbursement.java]: 72b94b28cc3026095e2f0bdc0755e4fc1a00aa2f

**Review by xenditjoshjimenez — APPROVED** [review]

**Review by itzpranz — APPROVED** [review]
LGTM

**Review by hecharles — APPROVED** [review]

File: README.md [hunk 11]
```diff
@@ -315,6 +322,123 @@ Disbursement disbursement = Disbursement.getById("EXAMPLE_ID");
...
+#### Get a PHP disbursement by reference ID
+
+```java
+/* Without client */
+DisbursementPHP disbursement = DisbursementPHP.getPHPByReferenceId("EXAMPLE_ID");
+/* With client */
+DisbursementPHP disbursement = xenditClient.disbursementPHP.getPHPByReferenceId("EXAMPLE_ID");
+```
+
+#### Get a PHP disbursement by ID
```
Comment (xen-HendryZheng) [inline README.md:433]: Please change all PHP to JAVA.

**Review by xen-HendryZheng — APPROVED** [review]

**Review by xen-HendryZheng — APPROVED** [review]

**Pull request merged**